ui/src/alerting/actions/checks.ts

```diff
@@ -213,9 +214,12 @@ export const saveCheckFromTimeMachine = () => async (
     } else {
       throw new Error(resp.data.message)
     }
-  } catch (e) {
-    console.error(e)
-    dispatch(notify(copy.createCheckFailed(e.message)))
```

Comment:
maybe keep the notification to user that create check did in fact fail?

Reply:
got it!